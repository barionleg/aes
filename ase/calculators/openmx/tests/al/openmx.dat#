DM.UseSaveDM  True
System.CurrentDirectory  /group1/schinavro/miniconda3/envs/py/lib/python3.6/site-packages/ase/calculators/openmx/tests/al.
System.Name  openmx
DATA.PATH  /group1/schinavro/libCalc/openmx3.8/DFT_DATA13
level.of.stdout  1
level.of.fileout  1
Species.Number  2
<Definition.of.Atomic.Species
Al Al8.0-s4p4d2 Al_PBE13
O O6.0-s3p3d2 O_PBE13
Definition.of.Atomic.Species>

Atoms.UnitVectors.Unit  AU
<Atoms.UnitVectors
  0.000000000000000   4.251883474302578   4.251883474302578 
  4.251883474302578   0.000000000000000   4.251883474302578 
  4.251883474302578   4.251883474302578   0.000000000000000 
Atoms.UnitVectors>


Atoms.Number  2
Atoms.SpeciesAndCoordinates.Unit  Ang
<Atoms.SpeciesAndCoordinates
   1   Al    0.00000000000000    0.00000000000000    0.00000000000000     1.5     1.5
   2    O    2.25000000000000    0.00000000000000    0.00000000000000     3.0     3.0
Atoms.SpeciesAndCoordinates>

scf.restart    on
scf.XcType  GGA-PBE
scf.energycutoff  150.0
scf.maxIter  300
scf.EigenvalueSolver  band
scf.Kgrid  2 2 2
scf.Mixing.Type  Rmm-Diis
scf.fixed.grid    -1.91334756343616   -4.03928930058745   -4.03928930058745
scf.criterion  1e-06


MD.Current.Iter   2
