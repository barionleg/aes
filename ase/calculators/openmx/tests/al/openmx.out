***********************************************************
***********************************************************

  This calculation was performed by OpenMX Ver. 3.8
  using 1 MPI processes and 1 OpenMP threads.

  Mon Mar 19 19:21:39 2018

***********************************************************
***********************************************************

DM.UseSaveDM  True
System.CurrentDirectory  /group1/schinavro/miniconda3/envs/py/lib/python3.6/site-packages/ase/calculators/openmx/tests/al.
System.Name  openmx
DATA.PATH  /group1/schinavro/libCalc/openmx3.8/DFT_DATA13
level.of.stdout  1
level.of.fileout  1
Species.Number  2
<Definition.of.Atomic.Species
Al Al8.0-s4p4d2 Al_PBE13
O O6.0-s3p3d2 O_PBE13
Definition.of.Atomic.Species>

Atoms.UnitVectors.Unit  Ang
<Atoms.UnitVectors
    0.000000000000000     2.250000000000000     2.250000000000000 
    2.250000000000000     0.000000000000000     2.250000000000000 
    2.250000000000000     2.250000000000000     0.000000000000000 
Atoms.UnitVectors>


Atoms.Number  2
Atoms.SpeciesAndCoordinates.Unit  Ang
<Atoms.SpeciesAndCoordinates
1 Al      0.000000000      0.000000000      0.000000000 1.5 1.5
2 O      2.250000000      0.000000000      0.000000000 3.0 3.0
Atoms.SpeciesAndCoordinates>

scf.restart  on
scf.XcType  GGA-PBE
scf.energycutoff  150.0
scf.maxIter  300
scf.EigenvalueSolver  band
scf.Kgrid  2 2 2
scf.Mixing.Type  Rmm-Diis
scf.fixed.grid  -1.91334756343616 -4.03928930058745 -4.03928930058745
scf.criterion  1e-06
MD.Current.Iter  1


***********************************************************
***********************************************************

  Required cutoff energy (Ryd) for 3D-grids = 150.0000
      Used cutoff energy (Ryd) for 3D-grids = 163.7790, 163.7790, 163.7790
  Num. of grids of a-, b-, and c-axes = 20, 20, 20

  Num.Grid1.    20
  Num.Grid2.    20
  Num.Grid3.    20


  Cell_Volume =    153.735462001529 (Bohr^3)
  GridVol     =      0.019216932750 (Bohr^3)
  Cell vectors (bohr) of the grid cell (gtv)
    gtv_a =  0.000000000000,  0.212594173715,  0.212594173715
    gtv_b =  0.212594173715,  0.000000000000,  0.212594173715
    gtv_c =  0.212594173715,  0.212594173715,  0.000000000000
    |gtv_a| =  0.300653563749
    |gtv_b| =  0.300653563749
    |gtv_c| =  0.300653563749

***********************************************************
***********************************************************


***********************************************************
***********************************************************
                  SCF history at MD= 1                    
***********************************************************
***********************************************************

   SCF=   1  NormRD=  1.000000000000  Uele= -4.205441119859
   SCF=   2  NormRD=  2.908761142111  Uele= -5.347825651683
   SCF=   3  NormRD=  2.638411305965  Uele= -5.253517518540
   SCF=   4  NormRD=  1.161953122666  Uele= -4.794270040495
   SCF=   5  NormRD=  0.342486576459  Uele= -4.571415004528
   SCF=   6  NormRD=  0.156296158228  Uele= -4.515180223680
   SCF=   7  NormRD=  0.113847431782  Uele= -4.506488519660
   SCF=   8  NormRD=  0.054675249813  Uele= -4.516463957613
   SCF=   9  NormRD=  0.012962654009  Uele= -4.500973425906
   SCF=  10  NormRD=  0.000146698567  Uele= -4.503547481416
   SCF=  11  NormRD=  0.000018971024  Uele= -4.503568978819
   SCF=  12  NormRD=  0.000007939861  Uele= -4.503574663921
   SCF=  13  NormRD=  0.000002457891  Uele= -4.503572206449
   SCF=  14  NormRD=  0.000000124324  Uele= -4.503572820719

*******************************************************
        Total energy (Hartree) at MD = 1        
*******************************************************

  Uele.         -4.503572820719

  Ukin.         12.447270234337
  UH0.        -288.230543950075
  UH1.           0.022831075529
  Una.         -15.508756855618
  Unl.           3.662395878257
  Uxc0.         -2.492567778987
  Uxc1.         -2.492567778987
  Ucore.       273.895755080991
  Uhub.          0.000000000000
  Ucs.           0.000000000000
  Uzs.           0.000000000000
  Uzo.           0.000000000000
  Uef.           0.000000000000
  UvdW           0.000000000000
  Utot.        -18.696184094554

  Note:

  Utot = Ukin+UH0+UH1+Una+Unl+Uxc0+Uxc1+Ucore+Uhub+Ucs+Uzs+Uzo+Uef+UvdW

  Uene:   band energy
  Ukin:   kinetic energy
  UH0:    electric part of screened Coulomb energy
  UH1:    difference electron-electron Coulomb energy
  Una:    neutral atom potential energy
  Unl:    non-local potential energy
  Uxc0:   exchange-correlation energy for alpha spin
  Uxc1:   exchange-correlation energy for beta spin
  Ucore:  core-core Coulomb energy
  Uhub:   LDA+U energy
  Ucs:    constraint energy for spin orientation
  Uzs:    Zeeman term for spin magnetic moment
  Uzo:    Zeeman term for orbital magnetic moment
  Uef:    electric energy by electric field
  UvdW:   semi-empirical vdW energy 

  (see also PRB 72, 045121(2005) for the energy contributions)



  Chemical potential (Hartree)      -0.136687671282

***********************************************************
***********************************************************
           Eigenvalues (Hartree) of SCF KS-eq.           
***********************************************************
***********************************************************

   Chemical Potential (Hatree) =  -0.13668767128188
   Number of States            =   9.00000000000005
   Eigenvalues
              Up-spin           Down-spin

   kloop=0
   k1=  -0.25000 k2=  -0.25000 k3=  -0.25000

    1   -0.92590730387839  -0.92590730387839
    2   -0.46335012028602  -0.46335012028602
    3   -0.35383680135998  -0.35383680135998
    4   -0.35383650025155  -0.35383650025155
    5   -0.21035158286459  -0.21035158286459
    6    0.01552886110795   0.01552886110795
    7    0.10716590780768   0.10716590780768
    8    0.10716609757734   0.10716609757734
    9    0.23099603040397   0.23099603040397
   10    0.24825919759716   0.24825919759716
   11    0.24826121664340   0.24826121664340
   12    0.46679292137910   0.46679292137910
   13    0.46679378450537   0.46679378450537
   14    0.66596855296214   0.66596855296214
   15    0.79016865480709   0.79016865480709
   16    0.89402523202079   0.89402523202079
   17    0.93992714573585   0.93992714573585
   18    0.93993116580353   0.93993116580353
   19    1.10690380856060   1.10690380856060
   20    1.10690895917315   1.10690895917315
   21    1.22902073380474   1.22902073380474
   22    1.22903507352011   1.22903507352011
   23    1.23648877189288   1.23648877189288
   24    1.68847955723339   1.68847955723339
   25    1.90072824050326   1.90072824050326
   26    1.90073175641912   1.90073175641912
   27    2.05523490760157   2.05523490760157
   28    2.05524232054543   2.05524232054543
   29    2.07134268626242   2.07134268626242
   30    2.41199467042742   2.41199467042742
   31    2.93554809185159   2.93554809185159
   32    2.96621202930926   2.96621202930926
   33    2.96621216800214   2.96621216800214
   34    3.54502811048970   3.54502811048970
   35    3.54505713466632   3.54505713466632
   36    3.60729885412243   3.60729885412243
   37    3.74054367826090   3.74054367826090
   38    3.74055690311654   3.74055690311654
   39    4.25243131249860   4.25243131249860
   40    5.55051309646715   5.55051309646715
   41    5.55053407140502   5.55053407140502
   42    6.54788491467170   6.54788491467170
   43    6.54790149341254   6.54790149341254
   44    6.70888859890839   6.70888859890839
   45    7.53924989268704   7.53924989268704
   46    7.53925143865861   7.53925143865861
   47    7.68721291774931   7.68721291774931
   48    9.01064843709647   9.01064843709647

   kloop=1
   k1=  -0.25000 k2=  -0.25000 k3=   0.25000

    1   -0.90028549730113  -0.90028549730113
    2   -0.49008840285329  -0.49008840285329
    3   -0.41809956077821  -0.41809956077821
    4   -0.37947165001094  -0.37947165001094
    5   -0.13602874887983  -0.13602874887983
    6   -0.04983374250893  -0.04983374250893
    7    0.00864734443575   0.00864734443575
    8    0.10309919848373   0.10309919848373
    9    0.17849743692344   0.17849743692344
   10    0.41521830762151   0.41521830762151
   11    0.41598014914905   0.41598014914905
   12    0.48712858635060   0.48712858635060
   13    0.56210178683511   0.56210178683511
   14    0.72467177227165   0.72467177227165
   15    0.77029784502485   0.77029784502485
   16    0.87633740094794   0.87633740094794
   17    0.97519324960899   0.97519324960899
   18    1.13157200629748   1.13157200629748
   19    1.17717336124225   1.17717336124225
   20    1.20019711976591   1.20019711976591
   21    1.27543659146217   1.27543659146217
   22    1.46207409832973   1.46207409832973
   23    1.57507336178730   1.57507336178730
   24    1.67371061176972   1.67371061176972
   25    1.76143779948649   1.76143779948649
   26    1.83147848445279   1.83147848445279
   27    2.03365105695670   2.03365105695670
   28    2.08403182877029   2.08403182877029
   29    2.10126558221740   2.10126558221740
   30    2.68433073648529   2.68433073648529
   31    2.79006218898514   2.79006218898514
   32    3.02118898831133   3.02118898831133
   33    3.09927054556659   3.09927054556659
   34    3.15758595115732   3.15758595115732
   35    3.48044747232382   3.48044747232382
   36    4.07133442804014   4.07133442804014
   37    4.09341306430058   4.09341306430058
   38    4.34185620117309   4.34185620117309
   39    4.60467400936372   4.60467400936372
   40    4.88554313596906   4.88554313596906
   41    5.33926128491697   5.33926128491697
   42    5.72173490461633   5.72173490461633
   43    6.39671706472000   6.39671706472000
   44    6.44777470396467   6.44777470396467
   45    6.59997884297916   6.59997884297916
   46    7.67675494611091   7.67675494611091
   47    8.11758232062530   8.11758232062530
   48    8.25276966971338   8.25276966971338

   kloop=2
   k1=  -0.25000 k2=   0.25000 k3=  -0.25000

    1   -0.90028575377403  -0.90028575377403
    2   -0.49008851823537  -0.49008851823537
    3   -0.41809839715861  -0.41809839715861
    4   -0.37947161175480  -0.37947161175480
    5   -0.13602929319535  -0.13602929319535
    6   -0.04983319909076  -0.04983319909076
    7    0.00864728708691   0.00864728708691
    8    0.10309951200812   0.10309951200812
    9    0.17849686534451   0.17849686534451
   10    0.41521813062610   0.41521813062610
   11    0.41597924690864   0.41597924690864
   12    0.48712802521380   0.48712802521380
   13    0.56210268527364   0.56210268527364
   14    0.72466944608041   0.72466944608041
   15    0.77029484651660   0.77029484651660
   16    0.87634052733290   0.87634052733290
   17    0.97519120636519   0.97519120636519
   18    1.13157199421477   1.13157199421477
   19    1.17717263878313   1.17717263878313
   20    1.20019644993044   1.20019644993044
   21    1.27543079423979   1.27543079423979
   22    1.46207172760764   1.46207172760764
   23    1.57507882682042   1.57507882682042
   24    1.67371705189289   1.67371705189289
   25    1.76144240251600   1.76144240251600
   26    1.83147825966784   1.83147825966784
   27    2.03364508070927   2.03364508070927
   28    2.08402911927658   2.08402911927658
   29    2.10126742603676   2.10126742603676
   30    2.68433140022657   2.68433140022657
   31    2.79005060702148   2.79005060702148
   32    3.02119001362036   3.02119001362036
   33    3.09927635469931   3.09927635469931
   34    3.15758454085961   3.15758454085961
   35    3.48044912041693   3.48044912041693
   36    4.07129994824303   4.07129994824303
   37    4.09339415869622   4.09339415869622
   38    4.34182593804708   4.34182593804708
   39    4.60467533601137   4.60467533601137
   40    4.88555902436787   4.88555902436787
   41    5.33924411243702   5.33924411243702
   42    5.72171260782295   5.72171260782295
   43    6.39672500242696   6.39672500242696
   44    6.44783102785250   6.44783102785250
   45    6.59997148588556   6.59997148588556
   46    7.67676985118477   7.67676985118477
   47    8.11756381404830   8.11756381404830
   48    8.25282663617508   8.25282663617508

   kloop=3
   k1=  -0.25000 k2=   0.25000 k3=   0.25000

    1   -0.90028559400752  -0.90028559400752
    2   -0.49008830145057  -0.49008830145057
    3   -0.41809912237231  -0.41809912237231
    4   -0.37947176550479  -0.37947176550479
    5   -0.13602943052684  -0.13602943052684
    6   -0.04983300370977  -0.04983300370977
    7    0.00864716874713   0.00864716874713
    8    0.10309967271495   0.10309967271495
    9    0.17849637403270   0.17849637403270
   10    0.41521937303774   0.41521937303774
   11    0.41598034403089   0.41598034403089
   12    0.48712764581457   0.48712764581457
   13    0.56210201688086   0.56210201688086
   14    0.72467160492685   0.72467160492685
   15    0.77029707818253   0.77029707818253
   16    0.87633977964830   0.87633977964830
   17    0.97519220400819   0.97519220400819
   18    1.13157243771211   1.13157243771211
   19    1.17717223417998   1.17717223417998
   20    1.20019613236831   1.20019613236831
   21    1.27543494587051   1.27543494587051
   22    1.46207256524967   1.46207256524967
   23    1.57507819073071   1.57507819073071
   24    1.67371447128057   1.67371447128057
   25    1.76143814853455   1.76143814853455
   26    1.83147840561491   1.83147840561491
   27    2.03364575284869   2.03364575284869
   28    2.08402989642915   2.08402989642915
   29    2.10126765692628   2.10126765692628
   30    2.68433175169817   2.68433175169817
   31    2.79005307196477   2.79005307196477
   32    3.02118819162409   3.02118819162409
   33    3.09927466090567   3.09927466090567
   34    3.15758329635990   3.15758329635990
   35    3.48044959920385   3.48044959920385
   36    4.07131833620484   4.07131833620484
   37    4.09339879479523   4.09339879479523
   38    4.34184948084184   4.34184948084184
   39    4.60467374983959   4.60467374983959
   40    4.88555808200607   4.88555808200607
   41    5.33925960488354   5.33925960488354
   42    5.72172752428060   5.72172752428060
   43    6.39672138507929   6.39672138507929
   44    6.44780301811489   6.44780301811489
   45    6.59997361163269   6.59997361163269
   46    7.67675537689585   7.67675537689585
   47    8.11756910095473   8.11756910095473
   48    8.25278371840990   8.25278371840990

***********************************************************
***********************************************************
                   Mulliken populations                    
***********************************************************
***********************************************************

  Total spin moment (muB)   0.000000000

                    Up spin      Down spin     Sum           Diff
      1   Al      1.350061356  1.350061356   2.700122712   0.000000000
      2    O      3.149938644  3.149938644   6.299877288   0.000000000

 Sum of MulP: up   =     4.50000 down          =     4.50000
              total=     9.00000 ideal(neutral)=     9.00000


  Decomposed Mulliken populations

    1   Al          Up spin      Down spin     Sum           Diff
            multiple
  s           0    0.587659386  0.587659386   1.175318771   0.000000000
   sum over m      0.587659386  0.587659386   1.175318771   0.000000000
  s           1   -0.055442896 -0.055442896  -0.110885792   0.000000000
   sum over m     -0.055442896 -0.055442896  -0.110885792   0.000000000
  s           2    0.024451462  0.024451462   0.048902925   0.000000000
   sum over m      0.024451462  0.024451462   0.048902925   0.000000000
  s           3   -0.000077206 -0.000077206  -0.000154411   0.000000000
   sum over m     -0.000077206 -0.000077206  -0.000154411   0.000000000
   sum over m+mul  0.556590746  0.556590746   1.113181493   0.000000000
  px          0    0.175533563  0.175533563   0.351067125   0.000000000
  py          0    0.175533261  0.175533261   0.351066522   0.000000000
  pz          0    0.175538055  0.175538055   0.351076111   0.000000000
   sum over m      0.526604879  0.526604879   1.053209758   0.000000000
  px          1    0.007003653  0.007003653   0.014007306   0.000000000
  py          1    0.007003490  0.007003490   0.014006980   0.000000000
  pz          1    0.007002609  0.007002609   0.014005218   0.000000000
   sum over m      0.021009752  0.021009752   0.042019504   0.000000000
  px          2    0.002744224  0.002744224   0.005488449   0.000000000
  py          2    0.002744468  0.002744468   0.005488936   0.000000000
  pz          2    0.002743517  0.002743517   0.005487035   0.000000000
   sum over m      0.008232210  0.008232210   0.016464419   0.000000000
  px          3    0.002525298  0.002525298   0.005050596   0.000000000
  py          3    0.002525304  0.002525304   0.005050609   0.000000000
  pz          3    0.002525328  0.002525328   0.005050657   0.000000000
   sum over m      0.007575931  0.007575931   0.015151861   0.000000000
   sum over m+mul  0.563422771  0.563422771   1.126845542   0.000000000
  d3z^2-r^2   0    0.065022416  0.065022416   0.130044832   0.000000000
  dx^2-y^2    0    0.065023713  0.065023713   0.130047426   0.000000000
  dxy         0    0.028184501  0.028184501   0.056369002   0.000000000
  dxz         0    0.028184953  0.028184953   0.056369905   0.000000000
  dyz         0    0.028185085  0.028185085   0.056370171   0.000000000
   sum over m      0.214600668  0.214600668   0.429201336   0.000000000
  d3z^2-r^2   1    0.009607826  0.009607826   0.019215651   0.000000000
  dx^2-y^2    1    0.009607358  0.009607358   0.019214716   0.000000000
  dxy         1   -0.001256184 -0.001256184  -0.002512369   0.000000000
  dxz         1   -0.001255973 -0.001255973  -0.002511946   0.000000000
  dyz         1   -0.001255856 -0.001255856  -0.002511712   0.000000000
   sum over m      0.015447170  0.015447170   0.030894341   0.000000000
   sum over m+mul  0.230047839  0.230047839   0.460095677   0.000000000

    2    O          Up spin      Down spin     Sum           Diff
            multiple
  s           0    0.982074026  0.982074026   1.964148052   0.000000000
   sum over m      0.982074026  0.982074026   1.964148052   0.000000000
  s           1   -0.043032231 -0.043032231  -0.086064463   0.000000000
   sum over m     -0.043032231 -0.043032231  -0.086064463   0.000000000
  s           2    0.000043161  0.000043161   0.000086323   0.000000000
   sum over m      0.000043161  0.000043161   0.000086323   0.000000000
   sum over m+mul  0.939084956  0.939084956   1.878169911   0.000000000
  px          0    0.723748277  0.723748277   1.447496554   0.000000000
  py          0    0.723747834  0.723747834   1.447495668   0.000000000
  pz          0    0.723747039  0.723747039   1.447494078   0.000000000
   sum over m      2.171243150  2.171243150   4.342486300   0.000000000
  px          1    0.010627879  0.010627879   0.021255758   0.000000000
  py          1    0.010627437  0.010627437   0.021254874   0.000000000
  pz          1    0.010625876  0.010625876   0.021251753   0.000000000
   sum over m      0.031881193  0.031881193   0.063762385   0.000000000
  px          2   -0.001621644 -0.001621644  -0.003243288   0.000000000
  py          2   -0.001621683 -0.001621683  -0.003243366   0.000000000
  pz          2   -0.001621810 -0.001621810  -0.003243621   0.000000000
   sum over m     -0.004865137 -0.004865137  -0.009730275   0.000000000
   sum over m+mul  2.198259205  2.198259205   4.396518410   0.000000000
  d3z^2-r^2   0   -0.000276442 -0.000276442  -0.000552884   0.000000000
  dx^2-y^2    0   -0.000276389 -0.000276389  -0.000552778   0.000000000
  dxy         0   -0.000370754 -0.000370754  -0.000741507   0.000000000
  dxz         0   -0.000370757 -0.000370757  -0.000741515   0.000000000
  dyz         0   -0.000370759 -0.000370759  -0.000741518   0.000000000
   sum over m     -0.001665100 -0.001665100  -0.003330201   0.000000000
  d3z^2-r^2   1    0.003123780  0.003123780   0.006247559   0.000000000
  dx^2-y^2    1    0.003124193  0.003124193   0.006248386   0.000000000
  dxy         1    0.002670452  0.002670452   0.005340903   0.000000000
  dxz         1    0.002670562  0.002670562   0.005341125   0.000000000
  dyz         1    0.002670597  0.002670597   0.005341195   0.000000000
   sum over m      0.014259584  0.014259584   0.028519168   0.000000000
   sum over m+mul  0.012594483  0.012594483   0.025188967   0.000000000

***********************************************************
***********************************************************
                    Dipole moment (Debye)                  
***********************************************************
***********************************************************

 Absolute D        9.37991152

                      Dx                Dy                Dz
 Total             -9.37991152        0.00006851        0.00001334
 Core              64.84329178        0.00000000        0.00000000
 Electron         -74.22320330        0.00006851        0.00001334
 Back ground       -0.00000000        0.00000000        0.00000000

***********************************************************
***********************************************************
       xyz-coordinates (Ang) and forces (Hartree/Bohr)  
***********************************************************
***********************************************************

<coordinates.forces
  2
    1    Al     0.00000   0.00000   0.00000   0.000000000855 -0.000000000447  0.000000000005
    2     O     2.25000   0.00000   0.00000  -0.000000000867  0.000000000444 -0.000000000008
coordinates.forces>

***********************************************************
***********************************************************
       Fractional coordinates of the final structure       
***********************************************************
***********************************************************

     1     Al     0.00000000000000   0.00000000000000   0.00000000000000
     2      O     0.50000000000000   0.50000000000000   0.50000000000000

***********************************************************
***********************************************************
               Computational Time (second)                 
***********************************************************
***********************************************************

   Elapsed.Time.       716.094

                               Min_ID   Min_Time       Max_ID   Max_Time
   Total Computational Time =     0      716.094          0      716.094
   readfile                 =     0        0.709          0        0.709
   truncation               =     0        0.000          0        0.000
   MD_pac                   =     0        0.000          0        0.000
   OutData                  =     0        0.144          0        0.144
   DFT                      =     0      712.843          0      712.843

*** In DFT ***

   Set_OLP_Kin              =     0       11.433          0       11.433
   Set_Nonlocal             =     0       10.274          0       10.274
   Set_ProExpn_VNA          =     0      106.145          0      106.145
   Set_Hamiltonian          =     0       18.162          0       18.162
   Poisson                  =     0        0.013          0        0.013
   Diagonalization          =     0        2.324          0        2.324
   Mixing_DM                =     0        0.272          0        0.272
   Force                    =     0      544.005          0      544.005
   Total_Energy             =     0        7.150          0        7.150
   Set_Aden_Grid            =     0        0.031          0        0.031
   Set_Orbitals_Grid        =     0        0.066          0        0.066
   Set_Density_Grid         =     0       12.946          0       12.946
   RestartFileDFT           =     0        0.005          0        0.005
   Mulliken_Charge          =     0        0.014          0        0.014
   FFT(2D)_Density          =     0        0.000          0        0.000
   Others                   =     0        0.003          0        0.003
